src/main/java/com/synopsys/arc/jenkins/plugins/ownership/util/OwnershipDescriptionHelper.java

```diff
@@ -73,21 +95,40 @@ public static String getOwnerEmail(@Nonnull OwnershipDescription descr) {
     }
     
     /**
-     * Gets e-mails of co-owners.
+     * Gets e-mails of co-owners (including owner).
      * @param descr Ownership description
-     * @return Comma-separated list of co-owner e-mails (may be empty)
+     * @param includeOwner Include owner to the list
+     * @return Set of co-owner emails
+     * @since TODO
      */
-    public static String getCoOwnerEmails(@Nonnull OwnershipDescription descr) {
-        StringBuilder coownerEmails=new StringBuilder(getOwnerEmail(descr));
+    public static Set<String> getCoOwnerEmailList(@Nonnull OwnershipDescription descr, boolean includeOwner) {
```

Comment:
So my plan is to switch to "primaryOwner" and "secondaryOwners". Should be better